### PR TITLE
Don't modify signature of 'Connection' constructor.

### PR DESCRIPTION
**Pull request by labkey-tchad**
#### Rationale
A previous update changed the signature of one of the 'Connection' constructors. This broke compatibility with some usages that were catching the `URISyntaxException` it claimed to throw.
This makes the constructors inconsistent but fixing those up is a job for another day.

#### Related Pull Requests
* #22 
* https://github.com/LabKey/labkey-api-jdbc/pull/17

#### Changes
* Make 'Connection' constructor match signature from v1.4.0

### CODE REVIEW

**Review by labkey-jeckels — COMMENTED** [review]
Is there a reason to reintroduce this exception instead of updating the JDBC side? As long as a change needs to be made, why not make them consistent instead?

**Comment by labkey-tchad** [comment]
> Is there a reason to reintroduce this exception instead of updating the JDBC side? As long as a change needs to be made, why not make them consistent instead?

Just trying to follow [semver](https://semver.org/) rules. Keep it backward compatible for 1.5.0 then do a wider fixup round for 2.0 at some point.
Maybe this library isn't used widely enough to conform that closely to semantic versioning rules.

**Comment by labkey-jeckels** [comment]
Makes sense. What about calling it 2.0.0? I know it's not a grand refactor or anything, but semver doesn't care.

File: labkey-client-api/src/org/labkey/remoteapi/query/ImportDataCommand.java [hunk 1]
```diff
@@ -297,7 +298,7 @@ public ImportDataCommand copy()
         return new ImportDataCommand(this);
     }
 
-    public static void main(String[] args) throws IOException
+    public static void main(String[] args) throws IOException, URISyntaxException
```
Comment (labkey-jeckels) [inline labkey-client-api/src/org/labkey/remoteapi/query/ImportDataCommand.java]: Is this edit needed now?

**Review by labkey-jeckels — APPROVED** [review]

**Comment by labkey-tchad** [comment]
Obviated by #26
